pkg/shell/nav.scss

```diff
@@ -389,15 +389,16 @@ $desktop: $phone + 1px;
 
 .pf-theme-dark {
   .pf-c-select__toggle {
-    //mobile
-    height: 100%;
-
     > * {
       color: var(--pf-global--Color--light-100);
     }
   }
 }
 
+.pf-c-select__toggle {
+  height: 100%;
```

Comment:
The original mobile comment is still relevant I suppose

Reply:
How come?

Reply:
I don't think the remaining css (color setting) is anyhow mobile related.

Reply:
I couldn't figure out its meaning

- The height didn't affect the mobile page

- About the color (That SVG arrow is affected)

It does affect the mobile page (in a significant area from the user's perspective)
![image](https://user-images.githubusercontent.com/39620234/233456897-b5e0295a-2524-489c-9eb6-7970c00ee895.png)

But it does also affect the desktop version (in a small area)
![image](https://user-images.githubusercontent.com/39620234/233457623-52c3e64b-6472-43cc-9a94-ba4028e82dee.png)

That might be the reason 'mobile' was mentioned, removed it anyway as it was not helpful, if there is something I missed I'll make the necessary changes.